source/standards/slis.html.md.erb

```diff
@@ -131,11 +131,10 @@ Observe mapped out the system components to complete the user journey “choose
 
 ![Choose a Grafana dashboard high-level system components](../images/choose-a-grafana-dashboard-high-level-system-components.png)
 
-Observe defined the SLI over 1 hour identifying:
+The team defined the SLI over last 1 hour where the service collects metrics, identifying:
```

Comment:
Not sure I understand 'last' in this sentence. Would it read better as "The team defined the SLI over the last hour the service collected metrics, identifying..." or have I misunderstood the meaning of the sentence?

Reply:
Hi, Steve, that does sound better. I will do a push to incorporate the comment.